release-notes/Release-Notes-RC2.md

```diff
@@ -0,0 +1,123 @@
+# Release Notes
+
+## RC2 released 5/16/2016
+
+Full details on this release can be read about in this blog post:
+** [the blog title](http://someurl) **
+
+Platform support has been expanded for RC2
```

Comment:
Can we point people to the getting started page or is this not done in Release notes?